### ISSUE TITLE
Mongo DB drivers

### ISSUE THREAD

**Issue by xnekv03** [eleftrik/laradhoc#9]
Hello,
thanks for the change but unfortunately I was unable to connect to Mongo with Laravel.
I forked your repository and modified
 
added some mongo specific environment variables
- https://github.com/xnekv03/laradhoc/blob/master/src/.env

added mogo connection config
- https://github.com/xnekv03/laradhoc/blob/master/src/config/database.php

Unfortunately there is still one docker problem that mongo drivers are missing. 
Steps to reproduce. 

- `git clone git@github.com:xnekv03/laradhoc.git`
- spin up docker containers
- run `php artisan migrate:fresh`
-  you will get an error `Unsupported driver [mongodb].` 

Thank you in advance

![error](https://i.imgur.com/cbczP9Z.png)

_Originally posted by @xnekv03 in https://github.com/eleftrik/laradhoc/issues/8#issuecomment-713162549_

**Comment by eleftrik** [comment]
@xnekv03 Thanks for your report and your patience.

If I connect to PHP-FPM container: ` docker exec -it laradhoc_php-fpm /bin/bash`

...MongoDB driver for PHP should be installed, because if I run `php -m | grep mongodb` I got `mongodb`.

As soon as possible, I'll try to setup a Laravel project to replicate your issue.

**Comment by eleftrik** [comment]
@xnekv03 I just setup a Laravel Project and I was able to run `php artisan migrate` both inside the PHP-FPM container and from my host (for my own convenience - I had to setup PHP 7.4, installed mongodb via PECL, etc.).

After running `composer require jenssegers/mongodb`, I added this to `config/database.php`:

```
'mongodb' => [
    'driver' => 'mongodb',
    'host' => env('DB_HOST', '127.0.0.1'),
    'port' => env('DB_PORT', 27017),
    'database' => env('DB_DATABASE', 'homestead'),
    'username' => env('DB_USERNAME', 'homestead'),
    'password' => env('DB_PASSWORD', 'secret'),
    'options' => [
        'database' => env('DB_DATABASE', 'admin'),
    ],
],
```


An extract of my `.env` file:

```
DB_CONNECTION=mongodb
DB_HOST=mongodb
DB_PORT=27017
DB_DATABASE="laradhoc"
DB_USERNAME="user"
DB_PASSWORD="password"
DB_AUTHENTICATION_DATABASE="laradhoc"

```

Note: if you're running artisan commands from your hosts (instead of connecting inside the PHP-FPM container), you must add `127.0.0.1 mongodb` to your `/etc/hosts` file.


After connecting to PHP-FPM container and running migrations:

![image](https://user-images.githubusercontent.com/6959298/96712573-a5f5a880-139f-11eb-9917-8f367a03fa9a.png)


I had my collections:

![image](https://user-images.githubusercontent.com/6959298/96712815-fa992380-139f-11eb-80c3-60c30706333d.png)

I hope this solves your issue!
